debug/CMakeLists.txt

```diff
@@ -7,13 +7,9 @@ set(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE CACHE STRING "" FORCE)
 set(CPACK_RPM_PACKAGE_LICENSE "MPLv2.0")
```

Comment:
Line 5-8 in general you want these in a top level or common cmake file (top level is fine in this case)